Tests.Integration.Presentation.Web/Contract/ItContractTest.cs

```diff
@@ -141,5 +141,31 @@ public async Task Can_Assign_And_Remove_DataProcessingRegistration()
             dto = await ItContractHelper.GetItContract(contract.Id);
             Assert.Empty(dto.DataProcessingRegistrations);
         }
+
+        [Fact]
+        public async Task Can_Delete_Contract_Advices()
```

Comment:
Jeg ville foretrække:

Delete_Contract_Removes_Associated_Advices

Samtidig ville jeg foretrække at du inddeler din kode i arrange act assert blokke.

Endeligt mangler validering for de andre entity types. Det må være lige til for de fleste har i forvejen en "create" og advice api er jo det samme på tværs